src/svg/DokumentIkkeTilgangIkon.js

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 function DokumentIkkeTilgangIkon(props) {
     return (
         <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" {...props}>
-            <g fill="#C6C2BF">
+            <g fill="#000">
```

Comment:
#3E3832 ? Eller hva nå enn tekst-fargen på siden er? 
Evt kan man bruke `currentColor` så vil ikonene få samme farge som teksten

Reply:
Mmmm, det kan sikkert være en god ide ja. Men da må vi se gjennom "alle" ikonene, for nå tror jeg de fleste er sorte